backend/mach.ml

```diff
@@ -35,6 +35,8 @@ type integer_operation =
 
 type float_comparison = Cmm.float_comparison
 
+type mutable_flag = Immutable | Mutable
```

Comment:
(I am not entirely sure why a new type is introduced;
the other file uses `Asttypes. mutable_flag`, with the
very same definition.)

Reply:
I think it's generally better not to pilfer one IR's types to use in another IR: it makes it easier to change in the future if one of the IRs needs to diverge.